idaes/core/solvers/petsc.py

```diff
@@ -414,13 +415,34 @@ def petsc_dae_by_time_element(
             interpret the trajectory data.
         trajectory_save_prefix (str or None): If a string is provided the
             trajectory data will be saved as gzipped json
+        between (list or tuple): List of time points to integrate between.  If
+            None use all time points in the model.  If the first or last time
+            point are not inclued, they will be added.
```

Comment:
> If the first or last time point are not inclued, they will be added.

1) inclued -> included

2) Requiring the first and last timepoints seems unnecessarily restrictive. Users may want to change the integrator settings for integrating during different time periods. If we don't want to support that now but come back to it later, we can raise a `NotImplementedError` if both those time points aren't included.

Reply:
I guess that's technically right, but the first point could be different for the initial conditions, so not including it may be looking for trouble.  The last point is probably safe to omit, but then you can't interpolate everything.  I went back and forth on this a bit couldn't really decide.

Reply:
I'd rather give users the flexibility to try something fancy if they want.

Reply:
I guess that fits with our general philosophy.

Reply:
Okay, made this change.